Formula/frum.rb

```diff
@@ -41,7 +41,7 @@ def install
     mkdir_p frum_dir/"versions/2.4.0"
     versions = shell_output("eval \"$(#{bin}/frum init)\" && frum versions").split("\n")
     assert_equal 2, versions.length
-    assert_match "2.4.0", versions[0]
-    assert_match "2.6.5", versions[1]
+    assert_includes versions, "  2.4.0"
```

Comment:
can we remove the spaces? 
```suggestion
    assert_includes versions, "2.4.0"
```

Reply:
I removed spaces!

Reply:
I see, I think it is fine then. Thanks!